handlers/Careers/components/CareersContent/index.js

```diff
@@ -38,6 +39,29 @@ var BENEFITS = [
 ];
 
 class CareersContent extends React.Component {
+  constructor() {
+      super();
+      this.state = { activePersonIndex: -1 };
```

Comment:
I’d like to keep all of this state internal to a popover component instead of bleeding it through the different use cases.